README.md

```diff
@@ -5,28 +5,30 @@ This js library will allow wallets and dapps to remind users that they own expir
 
 ## Usage
 
-
 ```js
 import { checkRenewal } from '@ensdomains/renewal'
 
 const userAddress = '0x123...'
 const referrerAddress = '0x234...'
 
 const {
-  hasExpiringDomains,
   numExpiringDomains,
   renewalUrl
  } = await checkRenewal(userAddress, referrerAddress)
 
-if(hasExpiringDomains){
+if(hasExpiringDomains > 0){
   return <a href={renewalUrl}>You have {numExpiringDomains} names needing renewal!</a>
 }
 ```
 
 - `renewalUrl` = `https:/app.ens.domains/address/0x123/renewal?referrer=0x345`
-- `numExpiringDomains` return number or `more than 100` if it has more than 100 names.
+- `numExpiringDomains` return only up to 100
+
+### Optional arguments
 
+-  `expirtyDate` = The default is set to 30 days from now
```

Comment:
expiryDate

Reply:
Also this isn't clear how this is filtering it. Maybe this should just be a number of days and you can work out the date from there

Reply:
expiryDate is just a date object, probably more flexible than setting number of dates as once can set the expiration time as well.